StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Intent.swift

```diff
@@ -92,15 +92,18 @@ enum Intent {
     }
 }
 
-// MARK: - IntentClientSecret
+// MARK: - RetrievableIntent
 
-/// An internal type representing a PaymentIntent or SetupIntent client secret
-enum IntentClientSecret {
+/// An internal type representing types of intents that can be retrieved from Stripe
+enum RetrievableIntent {
```

Comment:
Open to name suggestions here, but `IntentClientSecret` doesn't make sense any more.

Reply:
Can we delete this enum and replace it with `InitializationMode`?

Reply:
Ah yea good catch! Now with this we don't need to add new init's but rather make our existing internal ones public (under spi for now)